robot/EDA/resources/EDA.robot

```diff
@@ -135,11 +135,11 @@ Get Affiliation Mappings
     ${NS} =                 Get EDA namespace prefix
     [Arguments]             ${field_name}
     ${result} =             SOQL Query
-    ...                     SELECT ${NS}${field_name} FROM ${NS}Affl_Mappings__c
```

Comment:
I'm not sure about removing the NS variable from here @jonathanjayaraj @iskander-sfdc . It seems it was working fine previously.

Reply:
Correct. we'll need the ${NS} variable to add the EDA namespace if it is in a managed org. This was working previously.I'm about to try it in a regression org which is managed.

Reply:
Yes, this change fixes this test for a regression org which uses managed package